CONTRIBUTING.md

```diff
@@ -27,18 +30,19 @@ it. Once this is done it will proceed with actually running the plugin command.
 `opam monorepo pull` will fetch all the sources and properly set them up in a
 `duniverse/` folder.
 
-Note that this repository's lock file should be handled using the latest released
-version of `opam-monorepo`, not the dev version; i.e., you should use the latest version
-available on opam.
+Note that this repository's lock file should be handled using the version of
+`opam-monorepo` corresponding to the lockfile. This will always correspond
+to a released version. We do not use the dev version of opam-monorepo to prevent
```

Comment:
(For some reason I can't directly suggest changes as usual!)

`opam-monorepo` in monospace